client/scss/components/_list_card.scss

```diff
@@ -0,0 +1,105 @@
+//* @define list-card
+
+$list-card-number-size: 96px;
+
+.list-card {
```

Comment:
I would say that's a strange thing to have in the name then . - way to subjective.

Why not just call it a `numbered-list` or something that could then be included in something like a card?

Card is quite heavily used within web components...